src/metabase/public_settings.clj

```diff
@@ -9,112 +9,106 @@
             [metabase.public-settings.metastore :as metastore]
             [metabase.util.i18n :refer [available-locales-with-names set-locale]]
             [metabase.util.password :as password]
+            [puppetlabs.i18n.core :as i18n :refer [tru]]
```

Comment:
You can remove the :as i18n part, so you'd just have
`[puppetlabs.i18n.core :refer [tru]]`